deepchem/models/tensorgraph/layers.py

```diff
@@ -3112,3 +3113,196 @@ def none_tensors(self):
 
   def set_tensors(self, tensor):
     self.out_tensor, self.betas = tensor
+
+
+class PassThroughLayer(Layer):
+
+  def __init__(self, output_num, **kwargs):
+    self.output_num = output_num
+    super(PassThroughLayer, self).__init__(**kwargs)
+
+  def create_tensor(self, in_layers=None, set_tensors=True, **kwargs):
+    self.out_tensor = self.in_layers[0].out_tensors[self.output_num]
+
+
+class GraphCNNPoolLayer(Layer):
+  """
+  GraphCNNPool Layer from Robust Spatial Filtering with Graph Convolutional Neural Networks
```

Comment:
Would you mind adding a description of this layer at a high level and how it differs from our other pooling layers?